app/controllers/api/v1/base_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Api::V1::BaseController < ApplicationController
   rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+  skip_before_filter :verify_authenticity_token
```

Comment:
We were getting a CSRF error in the rails console when trying to post a new skill. Not sure why the "null session" in application controller wasn't taking care of this. If anyone knows how this might be a security vulnerability, I'd love to know!